src/e2e/java/uk/gov/hmcts/reform/sscs/functional/mya/SscsMyaBackendRequests.java

```diff
@@ -101,28 +101,49 @@ public CreatedCcdCase createOralCase(String emailAddress) throws IOException {
         );
     }
 
-    public void submitHearingEvidence(String hearingId, String description, String fileName) throws IOException {
+    public void uploadHearingEvidence(String hearingId, String fileName) throws IOException {
         HttpEntity data = MultipartEntityBuilder.create()
-            .setContentType(ContentType.MULTIPART_FORM_DATA)
-            .addBinaryBody("file",
-                Objects.requireNonNull(this.getClass().getClassLoader().getResourceAsStream(fileName)),
-                ContentType.IMAGE_PNG,
-                fileName)
-            .addTextBody("body", description)
-            .addTextBody("idamEmail", "mya-sscs-6920@mailinator.com")
-            .build();
+                .setContentType(ContentType.MULTIPART_FORM_DATA)
+                .addBinaryBody("file",
+                        Objects.requireNonNull(this.getClass().getClassLoader().getResourceAsStream(fileName)),
+                        ContentType.IMAGE_PNG,
```

Comment:
Is this type appropriate for audio video file?

Reply:
This method to upload only documents, we have separate method uploadSingleHearingEvidence for audioVideo file upload